packages/perseus/src/widgets/__tests__/radio_test.jsx

```diff
@@ -516,6 +581,50 @@ describe("single-choice question", () => {
         // Assert
         expect(screen.getAllByText("Incorrect (selected)")).toHaveLength(1);
     });
+
+    it.each([0, 1, 2, 3])(
+        "should display all rationales when static is true - test-id: %s",
+        (num) => {
+            // Arrange / Act
+            const staticQuestion = {
+                ...question,
+                widgets: {
+                    ...question.widgets,
+                    "radio 1": {
+                        ...question.widgets["radio 1"],
+                        static: true,
+                    },
+                },
+            };
+            renderQuestion(staticQuestion);
+
+            // Assert
+            expect(
+                screen.getByTestId(`perseus-radio-rationale-content-${num}`),
```

Comment:
I wonder if we can assert based on user-visible text instead of test ids?  Using test ids should be considered a last-resort. See https://testing-library.com/docs/queries/about#priority

Reply:
Was able to update this to check for visible text, but it's not able to search for the whole rationale as one phrase since it is split across multiple elements. I found some examples of functions we could use to locate the full text of each rationale, but I was having trouble getting them to work here. I decided just to search for the biggest or most unique chunk of text for each assertion. Please let me know if anyone knows of a way that works to search for text across elements. Thanks!

Reply:
I often do partial text matches for things like this. As long as the text is unique, I think it's fine. Thanks for adjusting the assertions.